{
  "_annotation": {
    "cve_id": "CVE-2024-29100",
    "reason": "Added CPE configurations because not yet analyzed by NVD.",
    "snapshot": "https://raw.githubusercontent.com/anchore/nvd-data-overrides/main/.snapshot/2024/CVE-2024-29100.json",
    "triage_notes": {
      "references": [
        "https://patchstack.com/database/vulnerability/ai-engine/wordpress-ai-engine-plugin-2-1-4-arbitrary-file-upload-vulnerability?_s_id=cve"
      ],
      "solutions": [
        "Update to 2.1.5 or a higher version."
      ]
    }
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:meowapps:ai_engine:*:*:*:*:*:wordpress:*:*",
                "matchCriteriaId": "4661F502-0776-5275-A8E7-99C7F6B33FF3",
                "versionEndExcluding": "2.1.5",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}